CRM/Report/Form/Contribute/Detail.php

```diff
@@ -160,7 +160,6 @@ public function __construct() {
           'total_amount' => array(
             'title' => ts('Amount'),
             'required' => TRUE,
-            'statistics' => array('sum' => ts('Amount')),
```

Comment:
@agh1 I agree about Cartesian product issues; that's my current beef with sum() in this usage. **But**, I think this line is what provides the "total contributions" statistic at the bottom of the report, and that a lot of people (who? "a lot of people" is all I know) rely on that bit of information in this report. Thoughts?

Reply:
Hey Allen,

I'm reviewing this as well by request. Will post below shortly with a couple of screenshots.